httpbakery/client.go

```diff
@@ -509,6 +517,18 @@ func RequestMacaroons(req *http.Request) []macaroon.Slice {
 	return mss
 }
 
+func decodeMacaroonSlice(value string) (macaroon.Slice, error) {
```

Comment:
// decodeMacaroonSlice decodes a base64-JSON-encoded slice of macaroons
// from the given string.

?